datasetinsights/evaluation_metrics/average_precision_2d.py

```diff
@@ -60,12 +60,11 @@ def update(self, mini_batch):
         """Update records per mini batch
 
         Args:
-            mini_batch (list(list)): a list which contains batch_size of
-            gt bboxes and pred bboxes pair in each image.
-            For example, if batch size = 2, mini_batch looks like:
-            [[gt_bboxes1, pred_bboxes1], [gt_bboxes2, pred_bboxes2]]
-            where gt_bboxes1, pred_bboxes1 contain gt bboxes and pred bboxes
-            in one image
+            mini_batch (list(list)): a list which contains batch_size of gt
+              bboxes and pred bboxes pair in each image. For example, if batch
```

Comment:
Why are these text indented by 2?

Reply:
It would let docstring know that it belongs to one parameter. [Reference](https://google.github.io/styleguide/pyguide.html).

Reply:
I was actually pointing out the indentation was not consistent. It should be 4 instead of 2.

Reply:
Fixed.